code/modules/mob/living/silicon/ai/latejoin.dm

```diff
@@ -37,6 +37,9 @@ var/global/list/empty_playable_ai_cores = list()
 			var/datum/game_mode/traitor/autotraitor/current_mode = ticker.mode
 			current_mode.possible_traitors.Remove(src)
 
+	// Ghost the current player and disallow them to return to the body
+	ghostize(0)
```

Comment:
This is a boolean, use FALSE

Reply:
Can do, but this is actually a set of binary flags. It is also used with 0 in other cases, such as normal ghosting through OOC > Ghost